app/routes/task.py

```diff
@@ -0,0 +1,184 @@
+import datetime
+from flask import Blueprint
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from dotenv import load_dotenv
+import requests
+import os
+
+
+bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
```

Comment:
Since `validate_model` is duplicated in task routes as well as goal routes, we could extract it into a separate route_helpers file and import it as needed!